### ISSUE TITLE
Missing tables in LearnPress 4.0.8

### ISSUE THREAD

**Issue by williamwithin** [LearnPress/learnpress#489]
I'm having issues with LearnPress 4.0.8 on a fresh WordPress 5.7.2 install using docker-compose. LearnPress was installed via the plugins menu in the WordPress admin. From the logs:

WordPress database error Table 'wordpress.wp_learnpress_section_items' doesn't exist
WordPress database error Table 'wordpress.wp_learnpress_order_items' doesn't exist

Looking in the database, these learnpress tables exist:

wp_learnpress_sections
wp_learnpress_sessions
wp_learnpress_user_item_results
wp_learnpress_user_itemmeta
wp_learnpress_user_items

Any suggestions about how to get this to work 'out of the box' would be appreciated.

**Comment by tungnxt89** [comment]
Hi,

We'll check and fix soon.

Thanks.

**Comment by jaydrogers** [comment]
I am getting similar issues when attempting to run database migrations after an upgrade.

### 🌍 Environment
- **WordPress Version:**  5.9.3 
- **LearnPress Version:** 4.1.6.3

### ⚠️ Error message
```
Step modify_tb_lp_section_items: Table 'mls_wordpress.wp_learnpress_section_items' doesn't exist
```
<img width="1749" alt="Screen Shot 2022-04-26 at 10 30 55" src="https://user-images.githubusercontent.com/3174134/165337340-a65a6d0c-b505-433c-b696-213ac12e0de0.png">

Any thoughts?

**Comment by tungnxt89** [comment]
Hi jaydrogers,

Are you run it success?

We don't why but the database of LP3, before you upgrade to LP4 is missing table learnpress_section_items.

If you can't fix that, please send us Admin info your site via email lewis@thimpress.com. We'll check and fix if it possible.

Thanks.

**Comment by tungnxt89** [comment]
> I'm having issues with LearnPress 4.0.8 on a fresh WordPress 5.7.2 install using docker-compose. LearnPress was installed via the plugins menu in the WordPress admin. From the logs:
> 
> WordPress database error Table 'wordpress.wp_learnpress_section_items' doesn't exist WordPress database error Table 'wordpress.wp_learnpress_order_items' doesn't exist
> 
> Looking in the database, these learnpress tables exist:
> 
> wp_learnpress_sections wp_learnpress_sessions wp_learnpress_user_item_results wp_learnpress_user_itemmeta wp_learnpress_user_items
> 
> Any suggestions about how to get this to work 'out of the box' would be appreciated.

We fixed it on the version 4.1.6.4.

Thanks.

**Comment by jaydrogers** [comment]
Upgrading to 4.1.6.5 fixed the issue for me, thanks!
